.github/workflows/aspire-leader-dev.yaml

```
@@ -2,7 +2,7 @@ name: Aspire-Deploy to dev
 on:
   push:
     branches:
-      - release-1.0.0
+      - 
```

actionlint (1.7.7)

string should not be empty

(syntax-check)